wwwroot/assets/scripts/global.js

```diff
@@ -0,0 +1,20 @@
+import indicatorsListMod from './indicatorsList';
```

Comment:
I see you are just ``` import * as g from 'global';```
I think it's better if you just export everything as a unique object.

Reply:
You mean mix indicatorsList.js and global.js in one common object to export?

Reply:
No I would export global as single object.
I know it's correct to export everything but If that's happening why not export an object with all those methods directly?